### PR TITLE
fixed cmdline coloring on windows

### LINKED ISSUE #7053: UnicodeEncodeError: 'latin-1' codec can't encode character '\U0001f916' in position 22: ordinal not in range(256)
Sentry Issue: [RASA-OPEN-SOURCE-7J](https://sentry.io/organizations/rasahq/issues/1912459064/?referrer=github_integration)

```
UnicodeEncodeError: 'latin-1' codec can't encode character '\U0001f916' in position 22: ordinal not in range(256)
  File "rasa/__main__.py", line 116, in main
    cmdline_arguments.func(cmdline_arguments)
  File "rasa/cli/scaffold.py", line 174, in run
    print_success("Welcome to Rasa! 🤖\n")
  File "rasa/shared/utils/cli.py", line 12, in print_success
    print_color(*args, color=rasa.shared.utils.io.bcolors.OKGREEN)
  File "rasa/shared/utils/cli.py", line 8, in print_color
    print(rasa.shared.utils.io.wrap_with_color(*args, color=color))
```

### PR DESCRIPTION
**Pull request by tmbo**
**Proposed changes**:
- fixed cmdline coloring on windows. fixes #7053 
- adds `colorama` as a windows only dependency (was already a dependency for development) 

**Status (please check what you already did)**:
- [x] ~added some tests for the functionality~
- [x] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: changelog/7053.bugfix.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixed cmdline coloring for windows cmdlines running an encoding other than utf-8.
```
Comment (wochinge) [inline changelog/7053.bugfix.md:1]:
```suggestion
Fixed command line coloring for windows command lines running an encoding other than `utf-8`.
```

File: rasa/shared/utils/cli.py [hunk 2]
```diff
@@ -5,7 +5,16 @@
 
 
 def print_color(*args: Any, color: Text):
```
Comment (wochinge) [inline rasa/shared/utils/cli.py:7]:
```suggestion
def print_color(*args: Any, color: Text) -> None:
```

File: pyproject.toml [hunk 3]
```diff
@@ -127,6 +127,7 @@ oauth2client = "4.1.3"
 regex = ">=2020.6,<2020.10"
 joblib = "^0.15.1"
 sentry-sdk = "^0.17.4"
+colorama = {version = "^0.4.4", platform = "windows"}
```
Comment (wochinge) [inline pyproject.toml:130]: Should we go for `sys_platform == 'win32'"` as this option is official documented?

File: rasa/shared/utils/cli.py [hunk 4]
```diff
@@ -5,7 +5,16 @@
...
-    print(rasa.shared.utils.io.wrap_with_color(*args, color=color))
+    output = rasa.shared.utils.io.wrap_with_color(*args, color=color)
+    try:
+        # colorama is used to fix a regression where colors can not be printed on
+        # windows. https://github.com/RasaHQ/rasa/issues/7053
+        from colorama import AnsiToWin32
```
Comment (wochinge) [inline rasa/shared/utils/cli.py:12]: do we need this for other print commands such as `raise_warning` as well?

and do we need a `init(wrap=False)` call as described in their [documentation](https://pypi.org/project/colorama/) for `init(wrap=True)`?

**Review by wochinge — APPROVED** [review]
Can also go into `2.0.x` 🚀

File: rasa/shared/utils/cli.py (hunk 4 above)
Comment (tmbo) [reply rasa/shared/utils/cli.py:12]: right good point, will add it to raise_warning as well. `init(...)` isn't needed, it doesn't do anything if wrap is set to `False` 😂

Comment (tmbo) [reply rasa/shared/utils/cli.py:12]: I looked into the `raise_Warning` call and we can't add this there: colorama uses system calls to color text on a tty stream, which this isn't. 

I also don't think it is an issue there as warnings should handle the encoding issues - but we can still intervene if we see issues popping up on sentry

Comment (wochinge) [reply rasa/shared/utils/cli.py]: sounds good. 

> init(...) isn't needed, it doesn't do anything if wrap is set to False 😂

I was wondering about that when reading their documentation 😂

**Pull request merged**